OrbitGl/App.cpp

```diff
@@ -172,6 +178,9 @@ void OrbitApp::OnTimer(const TimerInfo& timer_info) {
   if (timer_info.function_address() > 0) {
     const FunctionInfo& func =
         GetCaptureData().selected_functions().at(timer_info.function_address());
+    // const FunctionInfo* func =
```

Comment:
yes, leftover code. -> Removed it. 
I think this is fine during capture loading, since it only uses `selected_functions()` here, but I am going to test it again when I rebase

Reply:
Agree. As long as you only use selected_functions, this is fine!